transfers/transfer.py

```diff
@@ -11,12 +11,15 @@
 import base64
 import logging
 import logging.config  # Has to be imported separately
+import models
```

Comment:
Unfortunately using `absolute_import` didn't help.
I just tested with a virtualenv created with python 3 (3.4.3), and got an error `SystemError: Parent module '' not loaded, cannot perform relative import` when using `from . import models` However `import models` seem to work fine. So for me `import models` is working for python 2 and 3, when calling the script as the examples in the README. 
Do you run the tools in a different way?